.github/workflows/integration.yaml

```diff
@@ -0,0 +1,90 @@
+#TODO (ocobleseqx) It cannot be executed with the default GITHUB_TOKEN, requires a PAT but I cannot create one to use in an equinix-labs repo
```

Comment:
I'm curious, would this work with a GitHub App token or does it have to be a user token?

Reply:
I think it should work, but I also don't have access to install a GitHub app with the organization